src/LooCompare.jl

```diff
@@ -0,0 +1,134 @@
+import Base.show
+
+
+"""
+
+# LooCompare
+
+A struct containing the results of PsisLoo model comparisom.
+
+$(FIELDS)
+
+# Extended help
+
+# Fields
+  - `psis::Vector{PsisLoo}`                      : A vector of PsisLoo objects.
+  - `table::KeyedArray`                          : Comparison table.
+
+"""
+struct LooCompare
+    psis::Vector{PsisLoo}
+    table::KeyedArray
+end
+
+"""
+
+# loo_compare
+
+Construct a PsisLoo comparison table for loglikelihood matrices.
+
+$(SIGNATURES)
+
+# Extended help
+
+### Required arguments
+```julia
+* `loglikelihoods::Vector{Array{Float64, 3}}`    : Vector of loglikelihood matrices
+```
+
+### Optional arguments
+```julia
+* `model_names=nothing`                          : Optional specify models
+* `sort_models=true`                             : Sort models
```

Comment:
Could you elaborate? Perhaps:  :Sort models according to loo in ascending order

Reply:
Definitely, I also had not included an example of the LooCompare table in `??LooCompare`